vizro-core/docs/pages/examples/your-examples.md

```diff
@@ -0,0 +1,30 @@
+# Examples from Vizro users
+
+This page lists videos, blog posts, and examples of Vizro usage in repositories on GitHub. We've curated the list so it is a snapshot of projects and content that Vizro users have created.
+
+If you have something you'd like us to include on the list, or spot something that we should include, let us know:
+
+* by emailing (made-up-vizro-email TO DO),
+* or you can [raise an issue](https://github.com/mckinsey/vizro/issues) on the Vizro repository,
+* better still, you can [make a PR to contribute](../explanation/contributing.md) to this page.
+
+
+!!! note
+
+    The Vizro team and QuantumBlack, AI by McKinsey, do not take responsibility for third party content. In curating the list below, we may inspect or test an example at time of inclusion, but cannot guarantee the content thereafter.
+
+## Videos
+
+* From [Charming Data on YouTube](https://www.youtube.com/@CharmingData):
+    * [Build Python Data Apps with Vizro Dash](https://www.youtube.com/watch?v=wmQ6_GZ0zSk)
+    * [Introduction to Vizro Actions - Plotly Dash](https://www.youtube.com/watch?v=bom-9275Cic&t=8s)
+    * [Use Dash AG Grid within a Vizro app](https://www.youtube.com/watch?v=YvtVcXwQw0E)
+
+## Blog posts
+
+## Examples on GitHub
+<!-- vale off -->
+* [Proof of concept example by viiviandias](https://github.com/viiviandias/poc-vizro/blob/main/brasil_stocks.ipynb)
+* [Amazon sales analysis by Bottleneck44](https://github.com/Bottleneck44/Amazon-Sales-Analysis/blob/main/Amazon-analysis.ipynb)
+* [Opioid interactive map by rmejia41](https://github.com/rmejia41/vizro-opioid-interactive-app/blob/main/src/app.py)
+<!-- vale on -->
```

Comment:
Do we need to get their permission to mention them here? I guess not 😅 

A little bit of self-marketing 😄 : https://github.com/huong-li-nguyen/vizro-app-demos

I participate in data viz challenges privately using vizro 😄